src/GraphQl/Subscription/SubscriptionManager.php

```diff
@@ -42,14 +42,24 @@ public function __construct(private readonly CacheItemPoolInterface $subscriptio
 
     public function retrieveSubscriptionId(array $context, ?array $result, ?Operation $operation = null): ?string
     {
+
         /** @var ResolveInfo $info */
         $info = $context['info'];
         $fields = $info->getFieldSelection(\PHP_INT_MAX);
         $this->arrayRecursiveSort($fields, 'ksort');
         $iri = $operation ? $this->getIdentifierFromOperation($operation, $context['args'] ?? []) : $this->getIdentifierFromContext($context);
-        if (null === $iri) {
+        if (empty($iri)) {
```

Comment:
let's avoid a function call here if this can return an empty array we should fix the return type of the above functions.

Reply:
It also returns an empty string if you provide a value, but it's not a valid URI. That's why I made empty call instead of doing `null === $iri || '' === $iri || [] === $iri`.
Those 2 methods called above are part of a IdentifierTrait trait, means all call sides will need modification.